src/EventSubscriber/CaptchaCachedSettingsSubscriber.php

```diff
@@ -0,0 +1,41 @@
+<?php
+
+/**
+ * @file
+ * Contains \Drupal\captcha\EventSubscriber\CaptchaCachedSettingsSubscriber.
+ */
+
+namespace Drupal\captcha\EventSubscriber;
+
+use Drupal\Core\Config\ConfigCrudEvent;
+use Drupal\Core\Config\ConfigEvents;
+use Symfony\Component\EventDispatcher\EventSubscriberInterface;
+
+/**
+ * A subscriber clearing the cached definitions when saving captcha settings.
+ */
+class CaptchaCachedSettingsSubscriber implements EventSubscriberInterface {
+
+  /**
+   * Clearing the cached definitions whenever the settings are modified.
+   *
+   * @param \Drupal\Core\Config\ConfigCrudEvent $event
+   *   The Event to process.
+   */
+  public function onSave(ConfigCrudEvent $event) {
+    // Changing the Captcha settings means that any cached page might result in
```

Comment:
comment isn't really updated, it's not about cached pages, it's a bout the element info cache.